### PR TITLE
RSDK-9098: test go in module generation test

### PR DESCRIPTION
**Pull request by purplenicole730**
Tested workflow on this PR on commit `test go in workflow`. (results shown if you click the red X next to the commit hash)

NOTE FOR REVIEWERS: the other two failures in that commit are unrelated to this PR

### CODE REVIEW

File: cli/app.go [hunk 1]
```diff
@@ -1496,6 +1497,11 @@ After creation, use 'viam module update' to push your new module to app.viam.com
 					Name:  "generate",
 					Usage: "generate a new modular resource via prompts",
 					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:  moduleFlagLanguage,
+							Usage: "resource type to use in module",
+							Value: "python",
```
Comment (jckras) [inline cli/app.go:1503]: why do we have python here? I might be missing some context but is it acting as a default language?

**Review by jckras — COMMENTED** [review]
How do we test this?

File: cli/app.go (hunk 1 above)
Comment (purplenicole730) [reply cli/app.go:1503]: Yep, that's the default language if none is provided

**Comment by purplenicole730** [comment]
@jckras Sorry for not including testing in the description. Updated to explain how it was tested!

**Review by jckras — APPROVED** [review]
`test go in workflow` commit looks good LGTM

File: cli/app.go [hunk 2]
```diff
@@ -1496,6 +1497,11 @@ After creation, use 'viam module update' to push your new module to app.viam.com
 					Name:  "generate",
 					Usage: "generate a new modular resource via prompts",
 					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:  moduleFlagLanguage,
+							Usage: "resource type to use in module",
```
Comment (njooma) [inline cli/app.go:1502]: Update this usage text

**Review by njooma — CHANGES_REQUESTED** [review]

File: cli/app.go (hunk 2 above)
Comment (purplenicole730) [reply cli/app.go:1502]: Ah, a relic of copying and pasting the work from another PR. Thanks for catching!

**Review by njooma — APPROVED** [review]

**Pull request merged**